platform/mbed_sdk_boot.c

```diff
@@ -18,7 +18,9 @@
 #include <stdlib.h>
 #include <stdint.h>
 #include "cmsis.h"
-
+#if DEVICE_CRYPTOCELL
```

Comment:
This is SDK boot . The mbed OS 5 boot is in rtos folder (unified), see `mbed_boot.c` file

When and how this library should be initialized (why it was added to this boot sequence) ? Why trng_init does not initalizes this (is there better place to do init than adding this here) ?

Reply:
you are correct. This part was already removed as part of 98a9e91db34c8ebad5ad3276097c56ea7d416017

I probably returned it with my history changes. I'll remove it again